jbi/retry.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
+import logging
+from datetime import datetime, timedelta
+from os import getenv
+from time import sleep
+from typing import AsyncIterator
+
+import jbi.runner as runner
+from jbi.configuration import ACTIONS
+from jbi.queue import QueueItem, get_dl_queue
+
+CONSTANT_RETRY = getenv("CONSTANT_RETRY", "false") == "true"
+RETRY_TIMEOUT_DAYS = getenv("RETRY_TIMEOUT_DAYS", 7)
```

Comment:
If we're specifying these values in `jbi/environment.py`, we should probably access them with
```
settings = get_settings()
...
settings.constant_retry
...
settings.retry_timeout_days
```

However, if these settings only pertain to this script (and won't matter to the FastAPI app), maybe we can move them out of `environment.py` and access them like you have here. That, or build a new `BaseSettings` object so we get validation.

Reply:
I ran with moving them out of `environment.py` since they're only used in one place. This feels a little weird because we have 2 entry points into the same module. But we're sharing so much code I think it makes sense. Let me know if you think going another direction is better.